README.md

```diff
@@ -9,7 +9,7 @@ A python (3.5+) module that wraps pdftoppm and pdftocairo to convert PDF to a PI
 
 ### Windows
 
-Windows users will have to install [poppler for Windows](http://blog.alivate.com.au/poppler-windows/), then add the `bin/` folder to [PATH](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/).
+Windows users will have to install [poppler for Windows](http://blog.alivate.com.au/poppler-windows/), then add the `bin/` folder to [PATH](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/) or use poppler_path = r"C:\path\to\poppler-xx\bin" as an argument in ```convert_from_path``` function call.
```

Comment:
```suggestion
Windows users will have to install [poppler for Windows](http://blog.alivate.com.au/poppler-windows/), then add the `bin/` folder to [PATH](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/) or use `poppler_path = r"C:\path\to\poppler-xx\bin" as an argument` in `convert_from_path`.
```

Reply:
Just fixed the triple quoting that was not necessary here, if you add that it will be good to merge.